src/order/filter/pointwise.lean

```diff
@@ -129,6 +132,17 @@ protected def has_involutive_inv : has_involutive_inv (filter α) :=
 { inv_inv := λ f, map_map.trans $ by rw [inv_involutive.comp_self, map_id],
   ..filter.has_inv }
 
+localized "attribute [instance] filter.has_involutive_inv filter.has_involutive_neg" in pointwise
+
+@[simp, to_additive] protected lemma inv_le_inv_iff : f⁻¹ ≤ g⁻¹ ↔ f ≤ g :=
+⟨λ h, inv_inv f ▸ inv_inv g ▸ filter.inv_le_inv h, filter.inv_le_inv⟩
+
+@[to_additive] lemma inv_le_iff_le_inv : f⁻¹ ≤ g ↔ f ≤ g⁻¹ :=
+by rw [← filter.inv_le_inv_iff, inv_inv]
+
+@[simp, to_additive] lemma inv_le_self : f⁻¹ ≤ f ↔ f⁻¹ = f :=
```

Comment:
Is `le_self_inv`/`self_le_inv` worth having too? I'm not sure which name is better.

Reply:
I don't need it (yet). I think that we can add lemmas like this one when they're needed.